packages/api/cms-api/src/mailer/mailer.module.ts

```diff
@@ -14,6 +14,7 @@ export type MailerModuleConfig = {
     sendAllMailsTo?: string[];
     sendAllMailsBcc?: string[];
     disableMailLog?: boolean;
+    daysToKeepMailLog: number;
```

Comment:
Incompatible change.

Reply:
is it okay if I "just" change the changelog? or should we add this in two steps, first adding as optional, later as incompatible change switch to required?